src/rgw/rgw_sal_motr.cc

```diff
@@ -898,7 +900,9 @@ int MotrBucket::merge_and_store_attrs(const DoutPrefixProvider *dpp, Attrs& new_
   for (auto& it : new_attrs)
     attrs[it.first] = it.second;
 
-  return put_info(dpp, y, ceph::real_time());
+  // "put_info" is meant to update existing metadata, by
+  // explicitly passing false. Passing "yield" was incorrect
+  return put_info(dpp, false, ceph::real_time());
```

Comment:
// "put_info" second bool argument is meant to update existing metadata, which is not needed here, 
// so explicitly passing false.

No need to mention about yield here

Reply:
Done